### PR TITLE
Add directory purger before fixtures run

### PR DESCRIPTION
**Pull request by AdamKasp**
without purging catalog with invoices, we get an error of duplicated invoices while fixtures run

### CODE REVIEW

File: spec/Fixture/Listener/DirectoryPurgerListenerSpec.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+use Symfony\Component\Filesystem\Filesystem;
+
+final class DirectoryPurgerListenerSpec extends ObjectBehavior
+{
+    public function let(Filesystem $filesystem): void
+    {
+        $this->beConstructedWith($filesystem, 'path/to/invoices/');
+    }
+
+    public function it_remove_invoices_before_fixture_suite(Filesystem $filesystem, SuiteInterface $suite): void
```
Comment (GSadee) [inline spec/Fixture/Listener/DirectoryPurgerListenerSpec.php:28]:
```suggestion
    public function it_removes_invoices_before_fixture_suite(Filesystem $filesystem, SuiteInterface $suite): void
```

File: spec/Fixture/Listener/DirectoryPurgerListenerSpec.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        $filesystem->remove('path/to/invoices/')->shouldBeCalled();
+
+        $suiteEvent = new SuiteEvent($suite->getWrappedObject());
+
+        $this->beforeSuite($suiteEvent, []);
```
Comment (ernestWarwas) [inline spec/Fixture/Listener/DirectoryPurgerListenerSpec.php:34]:
```suggestion
        $this->beforeSuite(new SuiteEvent($suite->getWrappedObject()), []);
```

File: src/Fixture/Listener/InvoicesPurgerListener.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    public function beforeSuite(SuiteEvent $suiteEvent, array $options): void
+    {
+        $this->filesystem->remove($this->invoicesPath);
+    }
+
+    public function getName(): string
+    {
+        return 'invoices_directory_purger';
```
Comment (GSadee) [inline src/Fixture/Listener/InvoicesPurgerListener.php:34]: As you've changed the name of listener, you should also change it here:
```suggestion
        return 'invoices_purger';
```

**Review by GSadee — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Adam! :tada:

**Comment by kkevindev** [comment]
@GSadee release when? 😜

**Comment by GSadee** [comment]
Hi @kkevindev, I hope I will tag the new version even today 🤞🏻

**Pull request merged**